script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt

```diff
@@ -376,62 +389,64 @@ Suicide. Accident. Murder. Dissapearance.
 [sha:00154e36df256a3e12d8436aed49acb53f4bbea1]{
 -- Page 175, Offset 21796.
 -- その一連の記録は、呪われているとしか言いようがない。
-This series of records can only be described as cursed.
+The unbroken sequence recorded here can only be described as cursed.
 }
 [sha:24667e1edde0647d8f7acd13f8f4a01ca95bab3e]{
 -- Page 175, Offset 21797.
 -- 異常なのは、死因の大半が自死である事だ。
-The strange part is that the majority of the deaths were caused by suicide.
+Stranger still is that the majority of the deaths were suicides.
 }
 [sha:e416f0957d7a1fac05c59f48427f56e82c24488f]{
 -- Page 175, Offset 21798.
 -- 遠野家の当主たちは成人を迎える前か、その後に、自らの命を断っている―――
-The heads of the Tohno family took their own lives, either before they became an adult or afterward―――
+The heads of the Tohno family repeatedly took their own lives,
+ some before they even became an adult, others not long after―――
 }
 [sha:3b03b7bf880c82cb398cb7f246d0d2e3eda94e2b]{
 -- Page 176, Offset 21799.
 -- 視界から光が薄れていく。
-The light fades from my vision.
+My vision begins to grow dim.
 }
 [sha:db5f51e56d269113ae90ff7a233c8717ad3edcf1]{
 -- Page 176, Offset 21800.
 -- 後頭部にぐらぐらと血液が溜まっていくような感じ。
-It feels like the blood is messily accumulating in the back of my head.
+I feel as though all my blood is pooling in the back of my head.
 }
 [sha:c1ad643856b550bc9a61e1152e1f77990fa3c9e6]{
 -- Page 176, Offset 21801.
 -- 手足に力が入らなくなって、呼吸が段々と詰まっていく。
-My limbs are losing strength and my breathing is gradually getting more and more labored.
+My limbs have become unsteady,
+ and my breathing is gradually getting more and more labored.
 }
 [sha:2d7fb9dc3780ec7941a444f5db95aef5916483e2]{
 -- Page 176, Offset 21802.
 -- 俺は目眩と、イヤな予感から逃げるように、日記帳に目を移した。
-I move my eyes to the diary as if to escape the dizziness and unpleasant premonition.
+I try and escape the dizziness and unpleasant premonitions by turning my attention back to the diary.
 }
 [sha:0da4e4e174e52eb053444450fa8fe4e4de4f4bbc]{
 -- Page 176, Offset 21803.
 -- ページをめくる。冒頭の日付は2004年。
-I turn over the page. The date begins in 2004.
+I turn over the page. The date reads 2004.
 }
 [sha:3a660976579c8ee6a30fc41b365d98d3bee1145d]{
 -- Page 176, Offset 21804.
 -- ……槙久によって養子にとられた誰かが、人知れず病死した年だ。
-...It was the year when the one adopted by Makihisa died of an unknown illness.
+...The year the child Makihisa adopted died of an unknown illness.
 }
 [sha:27b9640b5e6ca73d8b8e43d0b12766ea0b9dc355]{
 -- Page 177, Offset 21805.
 -- 日記に書かれた<文字|テキスト>を追う。
-I follow the text written in the diary.
+I hungrily consume the text written in the diary.
```

Comment:
I think I see where you're getting the stronger sense of "follow", but I think the previous ver. was the correct interpretation.